hw/xbox/dsp/dsp_dma.c

```diff
@@ -77,13 +109,21 @@ static void dsp_dma_run(DSPDMAState *s)
             s->eol = true;
         }
 
+        /* Decode control word */
+        bool unk0 = (control >> 0) & 1; /* FIXME: What does this do? */
+        bool direction = control & NODE_CONTROL_DIRECTION;
+        uint32_t unk2 = (control >> 2) & 0x3;
+        bool unk4 = (control >> 4) & 1; /* FIXME: What does this do? */
```

Comment:
[I have documented unk4 on XboxDevWiki now](http://xboxdevwiki.net/DSP#Command_blocks).

So this can be renamed.